packages/nuxt3/src/core/nuxt.ts

```diff
@@ -104,7 +104,7 @@ export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
   const options = await loadNuxtConfig(opts)
 
   // Temporary until finding better placement for each
-  options.appDir = options.alias['#app'] = resolve(distDir, 'app')
+  options.appDir = options.alias['nuxt3/app'] /* for framework playground */ = options.alias['#app'] = resolve(distDir, 'app')
```

Comment:
Is it needed if we internally use `#app` ?

```suggestion
  options.appDir = options.alias['#app'] = resolve(distDir, 'app')
```

Reply:
Nope!